parsers/eiffel.c

```diff
@@ -149,6 +149,7 @@ static const keywordTable EiffelKeywordTable [] = {
 	{ "indexing",       KEYWORD_indexing   },
 	{ "infix",          KEYWORD_infix      },
 	{ "inherit",        KEYWORD_inherit    },
+	{ "insert",         KEYWORD_insert     },
```

Comment:
basically, it would mean the only change to this file would be ``` +	{ "insert",         KEYWORD_inherit     },``` and all the rest is left alone.